.github/workflows/integration-flows.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
               run: |
                   # Build, install CLI and verify it can run
                   npm install
-                  npm run build -w @nangohq/node -w @nangohq/shared -w nango
+                  npm run build -w @nangohq/node -w @nangohq/utils -w @nangohq/shared -w nango
```

Comment:
I think we could just `npm run ts-build`

Reply:
I think you forgot to change that, or put back a build command in utils/package.json, but I think using ts-build is safer

Reply:
Building everything seemed like overkill rather than just building what we need exactly. 

I’ll add back in the specific build command which was there originally in utils. I think it’s handy to have specific build commands in each package regardless

Reply:
https://github.com/NangoHQ/nango/pull/1916